packages/mdc-list/README.md

```diff
@@ -52,390 +52,132 @@ A basic list consists simply of the list itself, and list items taking up one li
-    <img class="mdc-list-item__graphic" src="/users/3/profile_pic.png"
-         width="56" height="56" alt="Picture of Peter Carlsson">
-    Peter Carlsson
-  </li>
-</ul>
-```
-
-#### Adding Metadata
-
-Metadata tiles can be added in a similar way to graphics. Place an element after the text
-with a `mdc-list-item__meta` class.
-
-```html
-<h2>Contacts</h2>
-<ul class="mdc-list">
-  <li class="mdc-list-item">
-    Janet Perkins
-    <a href="#" class="mdc-list-item__meta material-icons"
-       aria-label="Remove from favorites" title="Remove from favorites">
-      favorite
-    </a>
-  </li>
-  <li class="mdc-list-item">
-    Mary Johnson
-    <a href="#" class="mdc-list-item__meta material-icons"
-       aria-label="Add to favorites" title="Add to favorites">
-      favorite_border
-    </a>
+    <span class="mdc-list-item__text">
+      First-line text
+      <span class="mdc-list-item__secondary-text">
+        Second-line text
+      </span>
+    </span>
   </li>
   <li class="mdc-list-item">
-    Janet Perkins
-    <a href="#" class="mdc-list-item__meta material-icons"
-       aria-label="Add to favorites" title="Add to favorites">
-      favorite_border
-    </a>
+    <span class="mdc-list-item__text">
+      First-line text
+      <span class="mdc-list-item__secondary-text">
+        Second-line text
+      </span>
+    </span>
   </li>
 </ul>
 ```
 
-Supporting graphics and metadata can be combined easily. Check out the list demo for many examples of how
-tiles can be configured.
-
-> NOTE: If using controls such as a switch within a list tile, you may need to override
-> the width and height styles set on the tile element.
-
-### Using Ink Ripples for Interactive Lists
-
-MDC List supports adding ripples to `mdc-list-item` elements, for example in the case of a nav menu.
-To add ripples to lists, simply attach a ripple to each list item. Note that this can be easily done
-via `mdc-auto-init` when using [material-components-web](../material-components-web).
-
-```html
-<nav class="mdc-list">
-  <a href="/wifi" class="mdc-list-item" data-mdc-auto-init="MDCRipple">
-    <i class="material-icons mdc-list-item__graphic" aria-hidden="true">
-      network_wifi
-    </i>
-    Wi-Fi
-  </a>
-  <a href="/bluetooth" class="mdc-list-item" data-mdc-auto-init="MDCRipple">
-    <i class="material-icons mdc-list-item__graphic" aria-hidden="true">
-      bluetooth
-    </i>
-    Bluetooth
-  </a>
-  <a href="/data-usage" class="mdc-list-item" data-mdc-auto-init="MDCRipple">
-    <i class="material-icons mdc-list-item__graphic" aria-hidden="true">
-      data_usage
-    </i>
-    Data Usage
-  </a>
-</nav>
-<script>
-  mdc.autoInit();
-</script>
-```
-
-### List Dividers
-
-MDC List contains an `mdc-list-divider` class which can be used as full-width or inset
-subdivisions either within lists themselves, or standalone between related groups of content.
-
-To use within lists, simply add the `mdc-list-divider` class to a list item.
+#### List Items
```

Comment:
I'm inclined to remove this section and just put the paragraph describing list items under Single-line lists...but it's up to you

Reply:
No I think you're right. the item is part of the list, and this section is quite bare

Reply:
less is more